src/wasm-builder.h

```diff
@@ -221,8 +221,9 @@ class Builder {
     call->finalize();
     return call;
   }
+  template<typename T>
   CallIndirect* makeCallIndirect(Expression* target,
-                                 const std::vector<Expression*>& args,
```

Comment:
What else can this be?

Reply:
In this case it is an arena-allocated `ExpressionList`, used when I copy a `Call`'s arguments to a new `CallIndirect` expression. It would be more complicated and less efficient to copy the arguments out into a vector, just to have them copied back into an `ExpressionList`.